dotcom-rendering/src/layouts/DecideLayout.tsx

```diff
@@ -39,13 +39,26 @@ const DecideLayoutApps = ({ article, format, renderingTarget }: AppProps) => {
 	const notSupported = <pre>Not supported</pre>;
 	switch (format.display) {
 		case ArticleDisplay.Immersive: {
-			return (
-				<ImmersiveLayout
-					article={article}
-					format={format}
-					renderingTarget={renderingTarget}
-				/>
-			);
+			switch (format.design) {
```

Comment:
This now mirrors exactly what's going on in the web's decide layout logic elsewhere in the file